app/api/scraper.py

```diff
@@ -2,23 +2,23 @@
 
 from app import schemas
 from app.services.scrape import TwitterScraper
+from app.services.url_parser import parse_twitter_url
 
 router = APIRouter()
 
 
 @router.get("/check", response_model=schemas.CheckResponse, name="user:get-data")
 def user_info_check(url: str, scraper: TwitterScraper = Depends()):
-    # Validate input url
-    if url[:20] == "https://twitter.com/":
-        username = url.split("/")[3]
-    elif url[:12] == "twitter.com/":
-        username = url.split("/")[1]
-    elif url != "" and "/" not in url:
-        username = url
-    else:
```

Comment:
u forgot to remove same thing as this one in line 31->37, I can't comment on that :V

Reply:
@phhai-hcmut did u forget this?
on `user_detail_check()`